README.md

```diff
@@ -4,28 +4,20 @@ Mattermost is an open source, self-hosted Slack-alternative https://mattermost.o
 
 It's written in Golang and React and runs as a single Linux binary with MySQL or Postgres. Every month on the 16th [a new compiled version is released under an MIT license](https://about.mattermost.com/download/).
```

Comment:
Not directly related to this PR, but let's change the links on this line and the line above to `mattermost.com` rather than `mattermost.org` and `about.mattermost.com`